modules/claims_api/app/workers/claims_api/claim_establisher.rb

```diff
@@ -18,6 +19,21 @@ def perform(auto_claim_id) # rubocop:disable Metrics/MethodLength
       form_data = auto_claim.to_internal
       auth_headers = auto_claim.auth_headers
 
+      if auth_headers['va_eauth_birlsfilenumber'].blank? || auth_headers['va_eauth_pid'].blank?
+        # call MPI to get missing values
+        mpi = veteran_from_headers(auth_headers).mpi
+        auth_headers['va_eauth_pid'] = mpi.mvi_response.profile.participant_id
+        auth_headers['va_eauth_birlsfilenumber'] = mpi.mvi_response.profile.birls_id
```

Comment:
won't we get a no method error if mpi comes back invalid?

Reply:
Good point, just pushed a fix